lib/chef/provisioning/aws_driver/driver.rb

```diff
@@ -274,11 +296,23 @@ def allocate_machine(action_handler, machine_spec, machine_options)
           Chef::Log.debug('No key specified, generating a default one...')
           bootstrap_options[:key_name] = default_aws_keypair(action_handler, machine_spec)
         end
+
+        puts "#{machine_options.inspect}"
+
+        if machine_options[:is_windows]
+          Chef::Log.info "Setting winRM userdata..."
+          bootstrap_options[:user_data] = user_data
+        else
+          Chef::Log.info "Non-windows, not setting userdata"
```

Comment:
Can you make these two `.debug` statements?  They don't seem like actionable information *most* of the time.